addons/account/views/account_onboarding_templates.xml

```diff
@@ -0,0 +1,166 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <!-- ONBOARDING PANEL-->
+        <template inherit_id="web.onboarding_panel" id="onboarding_invoice_panel"
+                  name="account onboarding invoice" primary="True">
+            <xpath expr="//t[@t-set='image']" position="after">
+                <t>
+                    <t t-call="web.onboarding_step" name="company_step">
+                        <t t-set="title" t-value="'Company data'"/>
```

Comment:
Move to plain html this is wont be translatable


account_onboarding_views.xml

it's a mix of views action and templates